src/libman/Resources/Text.resx

```diff
@@ -126,6 +126,20 @@
   <data name="CleanCommandRemarks" xml:space="preserve">
     <value>    Deletes any folders that become empty after this operation. </value>
   </data>
+  <data name="ConfigCommand_Description" xml:space="preserve">
+    <value>Get or set configuration settings</value>
+  </data>
+  <data name="ConfigCommand_Examples" xml:space="preserve">
+    <value>    libman config http_proxy
```

Comment:
Nit: I'd personally prefer --get, --set and --remove. Did you run this by Ahmed?

Reply:
Talked to Ahmed.  We'll stick with the current implementation to closer align with NuGet's CLI syntax, but we can change this later if we get feedback.